### PR TITLE
added images another <p>

### PR DESCRIPTION
**Pull request by devonmoubry**

### CODE REVIEW

**Comment by devonmoubry** [comment]
placed divs around images for 965px for slider 
added <p> verbage for 965px additional

File: index.html [hunk 1]
```diff
@@ -22,7 +21,20 @@
         <p>
             Bringing you the best in bicycle gear, equipment and community since 1999!
         </p>
-        <img src="#" alt="most recent bike photo #1">
+        <div class="photos">
```
Comment (lovemecomputer) [inline index.html:24]: nice, we can use this for scrolling

**Comment by lovemecomputer** [comment]
looks good to me!

**Comment by briandanilo** [comment]
THIS IS THE GREATEST COMMIT OF ALL TIME!

**Pull request merged**